### ISSUE TITLE
Arrows are not very visible

### ISSUE THREAD

**Issue by jellypuno** [BroadcomMFD/cobol-control-flow#4]
When you try to check the flow of a commonly called procedure (such as print, close or error handling) the arrows are not very visible. It is kinda hard to follow the flow if i am just checking the chart. 

It would be great if I could move the boxes and re-arrange the flow chart so the arrows will not overlap but I am not sure if that is possible 😄 

![Capture](https://user-images.githubusercontent.com/45042840/89645908-14f05480-d8bb-11ea-981f-5a1884088d1b.JPG)

Also, The difference between a PERFORM.. THRU and PERFROM... UNTIL is not that obvious. I wish it could be distinct so that I know that there is a loop.

![image](https://user-images.githubusercontent.com/45042840/89645830-f38f6880-d8ba-11ea-8e7b-cd699bf0aeac.png)

**Comment by venkatzhub** [comment]
@jellypuno - keep those ideas flowing :)

As for the re-arranging the boxes - that is in the backlog. 

re: PERFORM THRU and PERFORM UNTIL...  as a user what do you think would make it obvious ? 

The intent of releasing this MVP was to get this kind of feedback!

**Comment by jellypuno** [comment]
> re: PERFORM THRU and PERFORM UNTIL... as a user what do you think would make it obvious ?

Maybe having the WS-EOF-YES inside a conditional box? (I mean the diamond shape 😆 )
At least in that way I would know that the procedure needs to meet a condition before it ends.

**Comment by venkatzhub** [comment]
@vibhor-bc  - probably this is something we need to talk to Tom on the UX part.
